src/applications/check-in/day-of/pages/Error.jsx

```diff
@@ -167,16 +168,34 @@ const Error = () => {
     <Wrapper pageTitle={header}>
       {alerts.map((alert, index) => (
         <div key={`alert-${index}`}>
-          <va-alert
-            show-icon
-            status={alert.type}
-            data-testid={`error-message-${index}`}
-            class={index !== 0 ? 'vads-u-margin-top--2' : ''}
-            uswds
-            slim
-          >
-            <div>{alert.message}</div>
-          </va-alert>
+          {alert.travelMessage ? (
+            <h2 data-testid="travel-message-heading">
+              {t('travel-reimbursement')}
+            </h2>
+          ) : (
+            <h2 data-testid="appointment-message-heading">
```

Comment:
This needs an error type check probably too, the designs don't have this heading for errors other than ones that originate from clicking check-in